roles.json

```
@@ -54,6 +54,25 @@
             ]
         }
     },
+    {
+        "role": "Strategic Planner",
+        "url": "strategic_planner",
+        "people": [
+            "All Coordination Committee Members"
+        ],
+        "role-head": "Strategic planner",
+        "responsibilities": {
+            "description": "Do long-term planning for the Astropy Project as a whole, including:",
+            "details": [
+                "Shape and provide a vision for next major steps in the Project, in consultation with the whole Astropy community.",
+                "Maintain a large-scale view of the project (both from developer and user perspectives).",
```

This feels vague, could we add something about to what end here?

```suggestion
                "Maintain an overview of all the project activities (both from developer and user perspectives).",
```